run_server.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+python3 ./matfyz_forum/manage.py makemigrations
+python3 ./matfyz_forum/manage.py migrate
+
+python3 ./matfyz_forum/manage.py runserver 0.0.0.0:8000
```

Comment:
This might be run directly from docker-compose. Also, record why the redirection of the port is necessary.

Reply:
The redirection is due to my laziness. Since we don't know the ip of django container beforehand i left django listen on all ips. This can be changed by setting a static ip in docker compose tho. But I don't see a reason to do so.

Reply:
The commentary was meaned as an appreciation of the Docker inner routing knowledge :)